apps/aecore/src/aec_peers.erl

```diff
@@ -40,6 +40,9 @@
 -endif.
 
 -include("peers.hrl").
+-define(MIN_PING_INTERVAL,   3000).
+-define(MAX_PING_INTERVAL, 120000).
+
 
 -type uri() :: http_uri:uri().
```

Comment:
Why keep this type around when it's not being used anymore?

Reply:
It's the next refactoring. It's still in quite a few places in this code.